apps/docs/src/examples/inputs/TagsSuggestionsEmpty.tsx

```diff
@@ -0,0 +1,106 @@
+import { useRef, useState } from "react";
+import { ClickAwayListener, Popper } from "@mui/material";
+import {
+  HvAdornment,
+  HvInput,
+  HvPanel,
+  HvTag,
+  HvTagProps,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+import { Add } from "@hitachivantara/uikit-react-icons";
+
+const colors = [
+  "Blue",
+  "Red",
+  "Green",
+  "Yellow",
+  "Purple",
+  "White",
+  "Black",
+  "Orange",
+  "Pink",
+  "Brown",
+  "Gray",
+  "Cyan",
+  "Magenta",
+  "Lime",
+  "Teal",
+  "Lavender",
+];
+
+export default function Demo() {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [selectedColors, setSelectedColors] = useState<string[]>([]);
+  const [open, setOpen] = useState(false);
+
+  const handleAddColor = (color: string) => {
+    setSelectedColors((prev) => {
+      const newColor =
+        typeof color === "string"
+          ? color
+          : ((color as HvTagProps).label as string);
+      const colorExists = prev.includes(newColor);
+      return colorExists ? prev : [...prev, newColor];
+    });
```

Comment:
What is this for? There is no `HvTagProps` - we're using an `HvInput`
```suggestion
    setSelectedColors((prev) => [...new Set([...prev, newColor])]);
```